test/unit/SearchResultsItems.test.jsx

```diff
@@ -81,6 +81,7 @@ describe('SearchResultsItems', () => {
         items: [],
         bibId: "b1234",
         id: "1234",
+        isArchiveCollection: false,
```

Comment:
There's a test in `ItemTable.test.js` that verifies that a heading is `Vol/Date`. Can you add a new one where that heading does not render but instead `Container` does? Otherwise, the rest of the update looks good.